llama_stack/apis/scoring_functions/scoring_functions.py

```diff
@@ -26,6 +26,10 @@ class Parameter(BaseModel):
 class LLMAsJudgeContext(BaseModel):
     judge_model: str
     prompt_template: Optional[str] = None
+    judge_score_regex: Optional[List[str]] = Field(
```

Comment:
We are making a strong assumption that a prompt_template is associated with a judge_score_regex to extract judge scores. (referring to [braintrust templates](https://github.com/braintrustdata/autoevals/blob/613f300483ca05d1f7a5b59e07e4e4a1edcfe107/templates/factuality.yaml#L19))